plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -715,22 +683,27 @@ private DataTypeTest DateTests()
                 dateOfLocalTimeChangeBackwardAtMidnightInSomeZone
                         .atStartOfDay().minusMinutes(1)).size() == 2);
 
-        return DataTypeTest.create()
+        SqlDataTypeTest dateTests = SqlDataTypeTest.create()
                 // before epoch
-                .addRoundTrip(dateDataType(), LocalDate.of(1952, 4, 3))
-                .addRoundTrip(dateDataType(), LocalDate.of(1970, 1, 1))
-                .addRoundTrip(dateDataType(), LocalDate.of(1970, 2, 3))
+                .addRoundTrip("DATE", "DATE '1952-04-03'", TIMESTAMP_MILLIS, "TIMESTAMP '1952-04-03 00:00:00.000'")
```

Comment:
In CreateAdInsertDataSetup, it will check if all inputs have explicit types, if not, it will check if sql executor is `TrinoSqlExecutor`. This test case contains oracleCreateAndInsert thus remove explicit type arguments here will get error message back.
https://github.com/trinodb/trino/blob/1a2ce32939b2ee5c125556b8c370de38abef2fe8/testing/trino-testing/src/main/java/io/trino/testing/datatype/CreateAndInsertDataSetup.java#L79-L89